### PR TITLE
adds shrubland & dry_shrubland

### PR DESCRIPTION
**Pull request by DeathShadez**

### CODE REVIEW

File: biomes/land/hills-small/subtropical/dry_shrubland.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+id: DRY_SHRUBLAND
+type: BIOME
+extends: [ EQ_HILLS, CARVING_LAND, BASE ]
+vanilla: minecraft:plains
+color: $biomes/colors.yml:DRY_SHRUBLAND
```
Comment (dfsek) [inline biomes/land/hills-small/subtropical/dry_shrubland.yml:5]: This key is not present in the colors.yml file

File: biomes/land/hills-small/subtropical/shrubland.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+id: SHRUBLAND
+type: BIOME
+extends: [ EQ_HILLS, CARVING_LAND, BASE ]
+vanilla: minecraft:plains
+color: $biomes/colors.yml:SHRUBLAND
```
Comment (dfsek) [inline biomes/land/hills-small/subtropical/shrubland.yml:5]: This key is not present in the colors.yml file

**Review by dfsek — CHANGES_REQUESTED** [review]
Looks good, just 2 color keys missing.

**Comment by DeathShadez** [comment]
Yep. Forgot to add push the colors.yml to github. Should be good now.

**Review by dfsek — APPROVED** [review]

**Pull request merged**